frontend/coprs_frontend/coprs/views/backend_ns/backend_general.py

```diff
@@ -354,6 +354,14 @@ def get_build_task(task_id):
     try:
         task = BuildsLogic.get_build_task(task_id)
         build_record = get_build_record(task)
+        if build_record is None:
+            jsonout = flask.jsonify(
+                {"msg": "Internal server error happened during searching"
+                        f" record for task {task_id} was not found"}
+            )
+            jsonout.status_code = 500
```

Comment:
Can we use error 400?  And I'm still not quite sure this is ideal, can't we raise this error directly in `get_build_task()`?

Reply:
OK, I used the already existing `get_copr_chroot_safe` method which returns 404, and changed the `except: return None` to re-raise - hopefully backend can handle this :D